bakery/pipeline-with-service.yml

```diff
@@ -0,0 +1,176 @@
+        file: pdf-spike/bakery/tasks/t-fetch-book.yml
+      - put: s3
+        on_failure:
+          put: event-update
+          params:
+            id: workflow-events-queued/id
+            status_id: "4" #Failed
+        on_success:
+          put: event-update
+          params:
+            id: workflow-events-queued/id
+            status_id: "3" #Processing
+
+  - name: assemble-book
+    plan:
+      - get: workflow-events-queued
+        trigger: true
+        passed: [fetch-book]
+      - get: pdf-spike
+      - task: look-up-book
+        file: pdf-spike/bakery/tasks/t-book-lookup.yml
+      - get: s3
+    #- get: cnx-recipes
+      - task: assemble-book-via-neb
+        file: pdf-spike/bakery/tasks/t-assemble-book.yml
+        on_failure:
+          put: event-update
+          params:
+            id: workflow-events-queued/id
+            status_id: "4" #Failed
+        on_success:
+          put: s3
+          params:
+            path: book
+
+  - name: bake-book
+    plan:
+      - get: workflow-events-queued
+        trigger: true
+        passed: [assemble-book]
+      - get: pdf-spike
+      - task: look-up-book
+        file: pdf-spike/bakery/tasks/t-book-lookup.yml
+      - get: s3
+      - get: cnx-recipes
+      - task: bake-book-via-easy-bake
+        file: pdf-spike/bakery/tasks/t-bake-book.yml
+        on_failure:
+          put: event-update
+          params:
+            id: workflow-events-queued/id
+            status_id: "4" #Failed
+        on_success:
+          put: s3
+          params:
+            path: book
+
+  - name: mathify-book
+    plan:
+      - get: workflow-events-queued
+        trigger: true
+        passed: [bake-book]
+      - get: pdf-spike
+      - task: look-up-book
+        file: pdf-spike/bakery/tasks/t-book-lookup.yml
+      - get: s3
+      - task: mathify-book-via-mathify
+        file: pdf-spike/bakery/tasks/t-mathify-book.yml
+        on_failure:
+          put: event-update
+          params:
+            id: workflow-events-queued/id
+            status_id: "4" #Failed
+        on_success:
+          put: s3
+          params:
+            path: book
+
+  - name: generate-pdf
+    plan:
+      - get: workflow-events-queued
+        trigger: true
+        passed: [mathify-book]
+      - get: pdf-spike
+      - task: look-up-book
+        file: pdf-spike/bakery/tasks/t-book-lookup.yml
+      - get: s3
+      - task: pdf-book-via-princexml
+        file: pdf-spike/bakery/tasks/t-build-pdf.yml
+      - put: s3-artifacts
+        params:
+          file: artifacts/*.pdf
+          acl: public-read
+          content_type: application/pdf
+        on_failure:
+          put: event-update
+          params:
+            id: workflow-events-queued/id
+            status_id: "4" #Failed
+        # on_success:
+        #   put: event-update
+        #   params:
+        #     id: workflow-events-queued/id
+        #     status_id: "5"
+        #     pdf_url: artifacts/pdf_url
```

Comment:
Does this still need to be commented?

Reply:
This part of the code does not work, and is not needed for this part PR but is needed for development moving forward.. short answer, shouldn;t even be there I will remove. Thank you!